src/wp-content/themes/twentytwenty/style.css

```
@@ -3601,6 +3601,7 @@ figure.wp-block-table.is-style-stripes {
 	line-height: 1.4;
 }
 
+.entry-content p img,
 .entry-content li img {
```

The ticket needed the 'changes-requested' keyword. 
- The specificity of `.entry-content p img` could be too high.
- Centered images added with Classic Editor need `display: block` to center properly.
- I closed ticket 57912 to address inline images in the Verse block at the same time.

One option with lower specificity:
```suggestion
html :where(.entry-content p img:not(.aligncenter)),
:where(.wp-block-verse) img,
.entry-content li img {
```